test/common/extensions/filters/http/platform_bridge/platform_bridge_filter_test.cc

```diff
@@ -1,3 +1,4 @@
+#include "test/mocks/event/mocks.h"
```

Comment:
Missing tests, right?

Reply:
But coverage isn't failing! ;)

In all seriousness, there's not a lot extra to cover here, but I am going to add coverage to cause the on-resume tests to be triggered by an "async callback", and basic coverage of callback state management. Also that bug you noted that's fixed in this PR.